docs-rtd/howto/manage-migrations.md

```diff
@@ -0,0 +1,101 @@
+(manage-migrations)=
+
+# Manage migrations
+
+The following document shows you what changes, if any, are needed to your Terraform plan when migrating a model(s) between Juju controllers and/or JAAS.
```

Comment:
I would put this as pre-migration checks. A step in the Migrate a model section in Manage models. See how we handle this in Juju: https://documentation.ubuntu.com/juju/3.6/howto/manage-models/#migrate-a-model